diffrax/_autocitation.py

```diff
@@ -359,17 +360,34 @@ def _virtual_brownian_tree(terms):
 
 @citation_rules.append
 def _space_time_levy_area(terms):
-    has_levy_area = lambda x: isinstance(x, AbstractBrownianPath) and x.levy_area != ""
+    has_levy_area = (
+        lambda x: isinstance(x, AbstractBrownianPath)
+        and x.levy_area != BrownianIncrement
+    )
     leaves = jtu.tree_leaves(terms, is_leaf=has_levy_area)
     if any(has_levy_area(leaf) for leaf in leaves):
-        _, levy_area_ref = _parse_reference_multi(VirtualBrownianTree)
-        return (
-            r"""
-% You are simulating Brownian motion using space-time Levy area, the formulae for which
-% were developed in:
+        return r"""
+% You are simulating Brownian motion using Levy area, the formulae for which
+% are due to:
+@misc{jelinčič2024singleseed,
+  title={Single-seed generation of Brownian paths and integrals
+  for adaptive and high order SDE solvers},
+  author={Andraž Jelinčič and James Foster and Patrick Kidger},
+  year={2024},
+  eprint={2405.06464},
+  archivePrefix={arXiv},
+  primaryClass={math.NA}
+}
+
+% and Theorem 6.1.6 of
+@phdthesis{foster2020a,
+  publisher = {University of Oxford},
+  school = {University of Oxford},
+  title = {Numerical approximations for stochastic differential equations},
+  author = {Foster, James M.},
+  year = {2020}
+}
```

Comment:
I think these citations deserve to also go `VirtualBrownianTree.__doc__`! :) You did a lot of work making this happen!

Reply:
Do you mean in the autocitation for the VBT? It is already in the doc of VBT, but I did add it to autocitation as well. I also added a little subsection on Levy areas in `docs/api/brownian/md`.

Reply:
What I mean is that I think these citations should appear in the docstring, and be parsed out with `_parse_reference_multi`.

Reply:
I wanted to include the line `and Theorem 6.1.6 of`, but I suppose it's not strictly necessary.